app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarActivity.java

```diff
@@ -534,40 +525,20 @@ protected void applyFilterCanceled(boolean val) {
         onResume();
     }
 
-    protected void applyFilterFitScreen(boolean val) {
```

Comment:
I was thinking if it might be a good idea to leave these funcions in but rename applyFilterFitScreen(val) to just fitScreen(). This method can then be called when the user selects new time limits.

Leave minHourHeight and maxHourHeight to default values, but set hourHeight so that the whole selected range is shown.

Selecting a time limit and then zooming out manually might be tedious...

Reply:
good idea, changed it ;)